[2025-01-05T17:50:25.282+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: insert_data.call_clear_all_data manual__2025-01-05T17:50:12.611557+00:00 [queued]>
[2025-01-05T17:50:25.290+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: insert_data.call_clear_all_data manual__2025-01-05T17:50:12.611557+00:00 [queued]>
[2025-01-05T17:50:25.290+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2025-01-05T17:50:25.497+0000] {taskinstance.py:2191} INFO - Executing <Task(SQLExecuteQueryOperator): call_clear_all_data> on 2025-01-05 17:50:12.611557+00:00
[2025-01-05T17:50:25.508+0000] {standard_task_runner.py:60} INFO - Started process 126 to run task
[2025-01-05T17:50:25.512+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'insert_data', 'call_clear_all_data', 'manual__2025-01-05T17:50:12.611557+00:00', '--job-id', '365', '--raw', '--subdir', 'DAGS_FOLDER/insert_data.py', '--cfg-path', '/tmp/tmpwpyl8czd']
[2025-01-05T17:50:25.515+0000] {standard_task_runner.py:88} INFO - Job 365: Subtask call_clear_all_data
[2025-01-05T17:50:25.594+0000] {task_command.py:423} INFO - Running <TaskInstance: insert_data.call_clear_all_data manual__2025-01-05T17:50:12.611557+00:00 [running]> on host 403e7aadd7ee
[2025-01-05T17:50:25.713+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='agushchin' AIRFLOW_CTX_DAG_ID='insert_data' AIRFLOW_CTX_TASK_ID='call_clear_all_data' AIRFLOW_CTX_EXECUTION_DATE='2025-01-05T17:50:12.611557+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-05T17:50:12.611557+00:00'
[2025-01-05T17:50:25.715+0000] {sql.py:276} INFO - Executing: CALL stage.clear_all_data()
[2025-01-05T17:50:25.726+0000] {base.py:83} INFO - Using connection ID 'postgres-db' for task execution.
[2025-01-05T17:50:25.738+0000] {base.py:83} INFO - Using connection ID 'postgres-db' for task execution.
[2025-01-05T17:50:25.750+0000] {sql.py:450} INFO - Running statement: CALL stage.clear_all_data(), parameters: None
[2025-01-05T17:50:25.784+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=insert_data, task_id=call_clear_all_data, execution_date=20250105T175012, start_date=20250105T175025, end_date=20250105T175025
[2025-01-05T17:50:25.847+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-01-05T17:50:25.891+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
